RemoveNthNode.java

```diff
@@ -0,0 +1,21 @@
+class Solution {
+    public ListNode removeNthFromEnd(ListNode head, int n) {
+        ListNode dummy = new ListNode (0);
```

Comment:
Good Job! this works. Please add comments and time and space complexities for this problem. Also please add the other 2 problem solution. You can contact me, the TA or the professor for any help.